src/popup/pages/Settings/index.tsx

```diff
@@ -0,0 +1,60 @@
+import React, { Component } from 'react';
+import { inject, observer } from 'mobx-react';
+import { Typography, Select, MenuItem, withStyles, WithStyles  } from '@material-ui/core';
+import { map } from 'lodash';
+
+import styles from './styles';
+import NavBar from '../../components/NavBar';
+import AppStore from '../../stores/AppStore';
+import { SessionLogoutInterval, sliArray } from '../../../models/SessionLogoutInterval';
+
+interface IProps {
+  classes: Record<string, string>;
+  store: AppStore;
+}
+
+@inject('store')
+@observer
+class Settings extends Component<WithStyles & IProps, {}> {
+  public render() {
+    const { classes } = this.props;
+
+    return(
+      <div className={classes.root}>
+        <NavBar hasBackButton title="Settings" />
+        <div className={classes.contentContainer}>
+          <div className={classes.fieldsContainer}>
+            <SliField {...this.props} />
+          </div>
+        </div>
+      </div>
+    );
+  }
+}
+
+const SliField = observer(({ classes, store: { sessionStore } }: any) => (
```

Comment:
add the type for a stateless component:
`const SliField: React.SFC<any>`

SFC is an alias for StatelessComponent